### PR TITLE
Adjusted the debug info according to issue #6701

### PR DESCRIPTION
**Pull request by ThomasMonroe314**
I saw the issue  #6701 and knew I could potentially make a PR for it, so here it is.

### CODE REVIEW

**Comment by sfan5** [comment]
You missed making most of the descriptive texts lowercase.

**Comment by paramat** [comment]
Not trivial.
:-1: This essentially undoes the request of #6666 which was to remove visual clutter and to fit on 2 lines. Non essential stuff was removed to make it shorter to fit on 2 lines. It's cleaner and readable to me.

**Comment by paramat** [comment]
This PR might be ok with some changes, just not as it is, let me test first.

**Comment by ThomasMonroe314** [comment]
Right, I had thought it looked a bit weird when it wasn't Capitalized

File: src/game.cpp [hunk 1]
```diff
@@ -4421,17 +4421,17 @@ void Game::updateGui(const RunStats &stats, f32 dtime, const CameraOrientation &
...
 		os << std::fixed
 			<< PROJECT_NAME_C " " << g_version_hash
-			<< ", FPS " << fps
+			<< ", FPS: " << fps
 			<< std::setprecision(0)
-			<< ", Drawtime " << drawtime_avg << "ms"
+			<< ", Drawtime: " << drawtime_avg << "ms"
 			<< std::setprecision(1)
-			<< ", Dtime jitter "
+			<< ", Drawtime jitter: "
```
Comment (paramat) [inline src/game.cpp]: Should be 'Dtime jitter' as this is used elsewhere and is more familiar, and shorter, also it seems 'dtime' is not 'drawtime'.

File: src/game.cpp [hunk 2]
```diff
@@ -4449,21 +4449,21 @@ void Game::updateGui(const RunStats &stats, f32 dtime, const CameraOrientation &
...
 		if (runData.pointed_old.type == POINTEDTHING_NODE) {
 			ClientMap &map = client->getEnv().getClientMap();
 			const INodeDefManager *nodedef = client->getNodeDefManager();
 			MapNode n = map.getNodeNoEx(runData.pointed_old.node_undersurface);
 
 			if (n.getContent() != CONTENT_IGNORE && nodedef->get(n).name != "unknown") {
-				os << ", Pointed " << nodedef->get(n).name
-					<< ", Param2 " << (u64) n.getParam2();
+				os << ", Pointed: " << nodedef->get(n).name
+					<< ", param2: " << (u64) n.getParam2();
```
Comment (paramat) [inline src/game.cpp]: Best keep this as 'Param2' for consistency.

**Comment by paramat** [comment]
Added line comments, with those done i think this might be ok.

**Comment by sfan5** [comment]
IMO all of the listed properties should be lowercase, having them uppercase just <i>doesn't look right</i>.

**Comment by paramat** [comment]
Except FPS and RTT, all lowercase is ok with me.

**Comment by paramat** [comment]
I'll merge this and edit on merge to add colons and make most lowercase.

**Comment by paramat** [comment]
screenshot_20171206_072704

^ Lowercase, no colons

screenshot_20171206_072931
 ^ Lowercase, colons

**Comment by paramat** [comment]
Actually i won't merge.
@ThomasMonroe314 i'll +1 this and merge it if colons are added and all variables are lowercase except FPS and RTT, as in my 2nd screenshot above.

**Comment by ThomasMonroe314** [comment]
ok thanks paramat I'll work on it.

**Comment by ThomasMonroe314** [comment]
@paramat Done.

File: src/game.cpp [hunk 3]
```diff
@@ -4421,17 +4421,17 @@ void Game::updateGui(const RunStats &stats, f32 dtime, const CameraOrientation &
...
 		os << std::fixed
 			<< PROJECT_NAME_C " " << g_version_hash
-			<< ", FPS " << fps
+			<< ", FPS: " << fps
 			<< std::setprecision(0)
-			<< ", Drawtime " << drawtime_avg << "ms"
+			<< ", drawtime: " << drawtime_avg << "ms"
 			<< std::setprecision(1)
-			<< ", Dtime jitter "
+			<< ", drawtime jitter: "
```
Comment (paramat) [inline src/game.cpp]: dtime jitter
But i can fix on merge.

**Comment by paramat** [comment]
:+1: but the error should be fixed on merge.

File: src/game.cpp (hunk 3 above)
Comment (ThomasMonroe314) [reply src/game.cpp]: oops forgot that one :P, I could fix it real quick.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
@sfan5 ok for you?

**Comment by paramat** [comment]
419799b8312244d6689b48e5fbdf8dd74b6cc7bd

**Pull request closed without merging**